### ISSUE TITLE
Doc: link for configuring Plone to use RelStorage is dead.

### ISSUE THREAD

**Issue by jimfulton** [zodb/relstorage#153]
http://shane.willowrise.com/archives/how-to-install-plone-with-relstorage-and-mysql/

fails

**Comment by jamadden** [comment]
Argh. It was working as of a week or so ago. It's a 500 error, so hopefully it's temporary. If it doesn't come back we can point to the [wayback machine](http://web.archive.org/web/20160622203030/http://shane.willowrise.com/archives/how-to-install-plone-with-relstorage-and-mysql/) or incorporate manually.

**Comment by mauritsvanrees** [comment]
@hathawsh That is your server, right?

**Comment by hathawsh** [comment]
Gah, Wordpress is failing only for anonymous users. It works when I'm logged in, so I didn't know it was broken. Checking it out.

**Comment by hathawsh** [comment]
Fixed.

**Comment by jamadden** [comment]
I'm not a Plone guy (we run one small Plone 5 installation, but mostly we're just straight Zope 3). Can anyone comment on how relevant the information in that link is today? It was written for Plone 3 and ZODB 3.7 in 2009. Some things are definitely not needed with newer ZODB versions (patching ZODB), some things are probably broken (the testing step, I believe will no longer work), some things may simply be outdated (specific MySQL configuration steps for a server with 256MB RAM) and some things I have no idea about (Plone's "fake-eggs"? Is that still a thing?).

Basically, what I'm wondering is if it's still relevant enough to be a main recommended link, or if it's aged enough that it should be demoted (noted as "historical") due to the chance for causing confusion. This is especially a concern since I don't think RelStorage 2.0 will work out of the box with a released version of Plone (until Plone can use newer BTrees/ZODB).

**Comment by hathawsh** [comment]
Yes, I consider it mostly historical now.

**Comment by mauritsvanrees** [comment]
The `fake-eggs` option was for the time that `Zope2` was not yet available as PyPI package. This was only needed on Plone 3. The first Plone 4.0 release uses the eggified Zope2 2.12.10 and a recent enough ZODB3 3.9.5 that does not need patches.

**Comment by jimfulton** [comment]
I really like the change you made to recast some of the configuration in terms of ZConfig. (I'm guessing that that was behind the excellent effort to rehabilitate the ZConfig docs.)

WRT Plone and current ZODB and RelStorage, I'm hopeful that the recent BTree change will allow Plone to get up to date.
